tasks/preload.js

```diff
@@ -126,8 +126,15 @@ module.exports = function (grunt) {
 					}
 				};
 
-				aVersionParts = options.compatVersion.split(".");
-				if (options.compatVersion === "edge" || aVersionParts[0] === 1 && +aVersionParts[1] >= 40) {
+				aVersionMatch = options.compatVersion.match(/^([0-9]+)\.([0-9]+)|edge$/)
```

Comment:
I think it would make more sense to exclude the `edge` check from the RegExp and do it separately beforehand.